src/api/index.js

```diff
@@ -230,25 +230,44 @@ export async function rejectApplication(mentor, reason) {
   return res.success;
 }
 
-export async function sendMentorshipRequest(mentor, mentorshipRequest) {
+export async function applyForMentorship(
+  mentor,
+  { myBackground, myExpectations, message }
+) {
+  const payload = {
```

Comment:
So why destructuring?

Reply:
for a more explicit and elegant syntax.

`const payload = {
    myBackground,
    myExpectations,
    message,
  };`
vs
`const payload = {
    myBackground: o.myBackground,
    ....
  };`
 or 
` function applyForMentorship(mentor,payload) {....}`